python/py-sip4/Portfile

```diff
@@ -66,7 +66,7 @@ if {${name} ne ${subport}} {
     checksums           rmd160 9709eda79d7e0a73eca82e25cbbfd0af6da074c8 \
                         sha256 b39d93e937647807bac23579edbff25fe46d16213f708370072574ab1f1b4211 \
                         size   1056384
-    revision            1
+    revision            2
```

Comment:
Revision should not be increased unless you've changed something that would be relevant for those users who were already able to install the port. It's explained in the MacPorts Guide.

Reply:
How do we know that no one installed it since it was broken? Notice, it did install, but installation was wrong.

Reply:
> Notice, it did install, but installation was wrong.

You're absolutely right, the revision needs to be increased.

Reply:
> I am not gonna insist here

Please do insist. I'm not infallible and in this case I did a poor drive-by review without understanding that the problem was not fixing a build failure. Fixing a build failure = don't increase revision; fixing wrong installed files = do increase revision.